rwengine/src/engine/SaveGame.cpp

```diff
@@ -1313,13 +1313,18 @@ bool SaveGame::getSaveInfo(const std::string& file, BasicState* basicState) {
 std::vector<SaveGameInfo> SaveGame::getAllSaveGameInfo() {
     using namespace boost::filesystem;
 
-    // TODO consider windows
+#if defined(RW_WINDOWS)
+    auto homedir = getenv("USERPROFILE");
+    const char gameDir[] = "\\My Documents\\GTA3 User Files";
```

Comment:
I may be wrong but using ``SHGetKnownFolderPath`` may be required for unicode usernames? Does the original game handle such paths?

Unless there is an issue including the Windows API it should be possible to call ``SHGetKnownFolderPath``. The main issue would be the wchar_t since OpenRW assumes UTF-8.